src/tests/ftest/server/daos_server_dump.py

```diff
@@ -54,6 +54,7 @@ def test_daos_server_dump_basic(self):
         Test Description: Test engine ULT stacks dump (basic).
 
         :avocado: tags=all
+        :avocado: tags=vm,daily_regression
```

Comment:
Minor, but our convention is to include when to run the functional test in the first tag line.
```suggestion
        :avocado: tags=all,daily_regression
        :avocado: tags=vm
```

Reply:
ok, but with "when to run the functional test" you mean separate the set of "timing" tags from the "config" tags, and put them first, right ?

Reply:
Right, put another way the format we follow is:
```
:avocado: tags=<when_to_run>      # e.g. 'all' + 'pr', 'daily_regression', or 'full_regression' (weekly)
:avocado: tags=<where_to_run>     # e.g. test stage used to run this test, e.g. 'vm', 'hw,small', 'hw,medium', or 'hw,large'
:avocado: tags=<feature_tags>     # tags used with the 'Features: <tag>' pragma
:avocado: tags=<test_tags>        # one or more tags used to select this test. Must include one unique tag (usually the full test method name)
```